web_form_banner/static/src/js/web_form_banner.esm.js

```diff
@@ -0,0 +1,216 @@
+odoo.define("web_form_banner.save_plus_load", function (require) {
+    var rpc = require("web.rpc");
+    var FormController = require("web.FormController");
+
+    var root = function (ctrl) {
+        return (ctrl && (ctrl.el || (ctrl.$el && ctrl.$el[0]))) || null;
+    };
+    var alive = function (ctrl) {
+        var r = root(ctrl);
+        return (
+            r &&
+            r.isConnected &&
+            !(typeof ctrl.isDestroyed === "function" && ctrl.isDestroyed())
+        );
+    };
+    var qsa = function (el, sel) {
+        return Array.from(el ? el.querySelectorAll(sel) : []);
+    };
+    var first = function () {
+        for (var i = 0; i < arguments.length; i++) {
+            var v = arguments[i];
+            if (v !== null && v !== "") return v;
```

Comment:
Do you have a reason of this change? I think this change should be reverted to be safe, otherwise.
```suggestion
            if (v != null && v !== "") return v;
```